SUMMARY.md

```diff
@@ -94,19 +95,25 @@
-      * [aqueduct_llm.dolly_v2_3b](api-reference/aqueduct-llm-reference/package-aqueduct-llm/aqueduct_llm.dolly_v2_3b.md)
-      * [aqueduct_llm.dolly_v2_7b](api-reference/aqueduct-llm-reference/package-aqueduct-llm/aqueduct_llm.dolly_v2_7b.md)
-      * [aqueduct_llm.llama_7b](api-reference/aqueduct-llm-reference/package-aqueduct-llm/aqueduct_llm.llama_7b.md)
-      * [aqueduct_llm.vicuna_7b](api-reference/aqueduct-llm-reference/package-aqueduct-llm/aqueduct_llm.vicuna_7b.md)
+      * [package aqueduct.resources](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/README.md)
+        * [aqueduct.resources.airflow](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.airflow.md)
+        * [aqueduct.resources.aws_lambda](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.aws_lambda.md)
+        * [aqueduct.resources.databricks](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.databricks.md)
+        * [aqueduct.resources.dynamic\_k8s](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.dynamic\_k8s.md)
+        * [aqueduct.resources.ecr](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.ecr.md)
+        * [aqueduct.resources.kubernetes](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.k8s.md)
+        * [aqueduct.resources.google\_sheets](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.google\_sheets.md)
+        * [aqueduct.resources.mongodb](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.mongodb.md)
+        * [aqueduct.resources.s3](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.s3.md)
+        * [aqueduct.resources.salesforce](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.salesforce.md)
+        * [aqueduct.resources.spark](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.spark.md)
+        * [aqueduct.resources.sql](api-reference/sdk-reference/package-aqueduct/package-aqueduct.resources/aqueduct.resources.sql.md)
+* [Aqueduct LLM Reference](api-reference/aqueduct-llm-reference/README.md)
+  * [package aqueduct_llm](api-reference/aqueduct-llm-reference/package-aqueduct-llm/README.md)
+    * [aqueduct_llm.dolly_v2_3b](api-reference/aqueduct-llm-reference/package-aqueduct-llm/aqueduct_llm.dolly_v2_3b.md)
+    * [aqueduct_llm.dolly_v2_7b](api-reference/aqueduct-llm-reference/package-aqueduct-llm/aqueduct_llm.dolly_v2_7b.md)
+    * [aqueduct_llm.llama_7b](api-reference/aqueduct-llm-reference/package-aqueduct-llm/aqueduct_llm.llama_7b.md)
+    * [aqueduct_llm.vicuna_7b](api-reference/aqueduct-llm-reference/package-aqueduct-llm/aqueduct_llm.vicuna_7b.md) 
```

Comment:
@saurav-c  This section needs to be reflected in `generate_docs.sh`

Reply:
I'll make a PR about including this.